src/main/java/Start.java

```diff
@@ -0,0 +1,35 @@
+import employeeServiceCRUD.controller.AddressController;
+import employeeServiceCRUD.controller.DepartmentController;
+import employeeServiceCRUD.controller.EmployeeController;
+import employeeServiceCRUD.controller.ModelController;
+import employeeServiceCRUD.repositories.AddressRepositoryIml;
+import employeeServiceCRUD.repositories.DepartmentRepositoryIml;
+import employeeServiceCRUD.repositories.EmployeeRepositoryIml;
+import employeeServiceCRUD.service.AddressService;
+import employeeServiceCRUD.service.DepartmentService;
+import employeeServiceCRUD.service.EmployeeService;
```

Comment:
employeeServiceCRUD is a bad name for the package. Can be employee.service.crud for example